api/database/factories/AssessmentResultFactory.php

```diff
@@ -60,8 +60,6 @@ public function withResultType(AssessmentResultType $type)
                 return [
                     'assessment_result_type' => $type->name,
                     'justifications' => $justifications,
-                    'other_justification_notes' => in_array(AssessmentResultJustification::FAILED_OTHER->name, $justifications) ?
-                        $this->faker->paragraph() : null,
```

Comment:
PR https://github.com/GCTC-NTGC/gc-digital-talent/pull/10083 removed this field but didn't quite catch it everywhere
I added this commit because it was a small change and more importantly gets rid of the below error allowing me to seed data again

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/8b99bd80-07e0-4e14-81bb-a47b272a0714)

Reply:
Nevermind https://github.com/GCTC-NTGC/gc-digital-talent/pull/10163